src/python/s2_common.i

```diff
@@ -127,8 +139,17 @@ std::vector<S2Polyline *> *out(std::vector<S2Polyline *> temp) {
   }
 }
 
+%inline %{
+  // mostly useful for testing
+  static PyObject *S2Point_FromRaw(double x, double y, double z) {
```

Comment:
Can this be a constructor?

Reply:
I had a go at that initially and ran into issues because of the [`S2Point` redefinition](https://github.com/google/s2geometry/blob/master/src/python/s2_common.i#L130-L137) happening. Suggestions welcome.

Reply:
Could you just add a comment that that's what's happening? Maybe we'll figure out how to clean it up later.

Reply:
done